orchagent/orch.h

```diff
@@ -152,7 +152,7 @@ class Orch
 
     /* Run doTask against a specific executor */
     virtual void doTask(Consumer &consumer) = 0;
-    virtual void doTask(NotificationConsumer &consumer) { }
+    virtual void doTask(NotificationConsumer &consumer, const std::string &name) { }
```

Comment:
>const std::string &name [](start = 56, length = 23)

can you add a comment on the name, what does this name use for?

Reply:
Comments added

Reply:
I've also renamed this parameter to more clear name